internal/k8s/advl4_controller.go

```diff
@@ -185,17 +185,17 @@ func (c *AviController) SetupNamespaceEventHandler(numWorkers uint32) {
 		UpdateFunc: func(old, cur interface{}) {
 			nsOld := old.(*corev1.Namespace)
 			nsCur := cur.(*corev1.Namespace)
-			if isNamespaceUpdated(nsOld, nsCur) {
```

Comment:
why are we removing this check?

Reply:
It was unnecessary as after checking this we again check for tenant annotation change. Also, we want to do retry ingress on change in tenant annotation as well as infrasettingname annotation